src/hooks/dialogmote/useForhandsvisningIntro.ts

```diff
@@ -0,0 +1,16 @@
+import { useNavBrukerData } from "@/data/navbruker/navbruker_hooks";
+import { createParagraph } from "@/utils/documentComponentUtils";
+
+export const useForhandsvisningIntro = () => {
+  const navBruker = useNavBrukerData();
+  const gjelder = createParagraph(
+    `Gjelder ${navBruker.navn}, f.nr. ${navBruker.kontaktinfo.fnr}`
+  );
```

Comment:
Hvor skriver vi f.nr. her egentlig? Er det fordi AG omtaler D-nummer som F-nummer?

Reply:
Da må du nesten spørre de som har utarbeidet tekstene ;) For meg er f.nr. er et mer kjent begrep enn d-nr.